### PR TITLE
ADD AT32F435VMT7  AT32F435RGT7 targets

### PR DESCRIPTION
**Pull request by shanggl**
this pr add two targets for test bf 4.5 prepare for  pr at32 's modify

### CODE REVIEW

File: configs/GREATMOUNTAINF435/config.h [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+#define BARO_I2C_INSTANCE               I2CDEV_3
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE            SPI2
+#define MAX7456_SPI_CS_PIN              PD5
+
+#define TIMER_PIN_MAPPING               TIMER_PIN_MAP( 1, LED_STRIP_PIN, 2, 7 ) \
```
Comment (haslinghuis) [inline configs/GREATMOUNTAINF435/config.h:116]: TIMER_PIN_MAPPING uses zero based index

File: configs/GREATMOUNTAINF435/config.h [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                                        TIMER_PIN_MAP( 8, MOTOR2_PIN, 2,  2 ) \
+                                        TIMER_PIN_MAP( 9, MOTOR3_PIN, 1,  1 ) \
+                                        TIMER_PIN_MAP(10, MOTOR4_PIN, 1,  3 )
+
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE     110
+#define DEFAULT_CURRENT_METER_SCALE     100
+
+#define DEFAULT_FEATURES (FEATURE_OSD |FEATURE_LED_STRIP)
```
Comment (haslinghuis) [inline configs/GREATMOUNTAINF435/config.h:132]:
```suggestion
#define DEFAULT_FEATURES (FEATURE_OSD | FEATURE_LED_STRIP)
```

File: configs/NEUTRONRCF435SE/config.h [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+#define BARO_I2C_INSTANCE               I2CDEV_2
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE            SPI2
+#define MAX7456_SPI_CS_PIN              PB12
+
+#define TIMER_PIN_MAPPING               TIMER_PIN_MAP( 1, LED_STRIP_PIN, 2, 7 ) \
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435SE/config.h:123]: As above

File: configs/NEUTRONRCF435SE/config.h [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+                                        TIMER_PIN_MAP( 8, MOTOR2_PIN, 2,  2 ) \
+                                        TIMER_PIN_MAP( 9, MOTOR3_PIN, 1,  1 ) \
+                                        TIMER_PIN_MAP(10, MOTOR4_PIN, 1,  3 )
+
+#define DEFAULT_CURRENT_METER_SOURCE    CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE    VOLTAGE_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SCALE     110
+#define DEFAULT_CURRENT_METER_SCALE     100
+
+#define DEFAULT_FEATURES (FEATURE_OSD |FEATURE_LED_STRIP)
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435SE/config.h:139]:
```suggestion
#define DEFAULT_FEATURES (FEATURE_OSD | FEATURE_LED_STRIP)
```

**Comment by ASDosjani** [comment]
@shanggl please fix the issues, I want to test these targets because I have the 2 FC.

**Review by nerdCopter — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

File: configs/NEUTRONRCF435SE/config.h [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           NEUTRONRCF435SE
+#define MANUFACTURER_ID      NERC
+
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACCGYRO_BMI270
+#define USE_ACCGYRO_ASM330LHH
```
Comment (haslinghuis) [inline configs/NEUTRONRCF435SE/config.h:32]: We have a new IMU: ASM330LHH

https://www.st.com/en/mems-and-sensors/asm330lhh.html

**Comment by ASDosjani** [comment]
Thank you @haslinghuis @nerdCopter

**Pull request merged**